cfme/tests/configure/test_access_control.py

```diff
@@ -562,8 +562,8 @@ def test_assign_user_to_new_group(appliance, group_collection):
 
 def _test_vm_provision(appliance):
     logger.info("Checking for provision access")
-    navigate_to(vms.Vm, 'VMsOnly')
-    vms.lcl_btn("Provision VMs")
+    view = navigate_to(vms.Vm, 'VMsOnly')
+    view.toolbar.lifecycle.item_enabled('Provision VMs')
```

Comment:
**QUESTION**
What's the purpose of this line?

Reply:
To check if 'Provision VMs' item is available in the Lifecycle dropdown, since we are testing provisioning permissions for the user.